packages/react/src/components/DatepickerNavigation/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+import IconButton from '../IconButton';
+
+const DatepickerNavigation = React.forwardRef(
+  ({ month, onPreviousClick, onNextClick }, ref) => {
+    return (
+      <div ref={ref} className='f-Datepicker--navbar'>
+        {month.toLocaleString('default', { month: 'long' })}{' '}
+        {month.getFullYear()}
+        <div className='f-Datepicker--actions'>
+          <IconButton
+            onClick={() => onPreviousClick()}
+            icon={Icon.ICONS.IconChevronLeft}
+          />
+          <IconButton
+            onClick={() => onNextClick()}
+            icon={Icon.ICONS.IconChevronRight}
+          />
+        </div>
+      </div>
+    );
+  },
+);
+
+DatepickerNavigation.displayName = 'DatepickerNavigation';
+
+DatepickerNavigation.propTypes = {
+  month: PropTypes.instanceOf(Date),
+  onPreviousClick: PropTypes.func,
+  onNextClick: PropTypes.func,
+};
+
+DatepickerNavigation.defaultProps = {
+  month: null,
+  onPreviousClick: () => {},
+  onNextClick: () => {},
+};
```

Comment:
Can be removed.

Also, if `month` is null the component would explode since you're trying to run methods on it. Maybe `if(!month) return null`